tuna/machine.py

```diff
@@ -319,21 +358,28 @@ def read_file(self, filename, byteread=False):
       ftp.getfo(filename, content_io)
       ret = content_io.getvalue()
       if not byteread:
-        ret = ret.decode()
+        ret_str = ret.decode()
+        return ret_str
```

Comment:
can you set ret as Any[str, bytes] and return ret, thus removing the extra ret_str and leave the code as it was to just return ret.

Reply:
I used Union[str, Bytes].